server/handlers/isomorphic-handler.js

```diff
@@ -159,9 +161,13 @@ exports.handleIsomorphicShell = async function handleIsomorphicShell(
         `<${assetHelper.assetPath("app.js")}>; rel=preload; as=script;`
       );
     }
+    const seoInstance = getSeoInstance(seo, config, "shell");
```

Comment:
passing hardcoded pageType `shell` to seo because we're doing the same on line 147 of this file